components/catalogs/back/src/Infrastructure/Persistence/FindOneAttributeByCodeQuery.php

```diff
@@ -36,6 +36,8 @@ public function execute(string $code): ?array
             'type' => $attribute->getType(),
             'scopable' => $attribute->isScopable(),
             'localizable' => $attribute->isLocalizable(),
+            'measurement_family' => $attribute->getMetricFamily(),
+            'default_measurement_unit' => $attribute->getDefaultMetricUnit(),
```

Comment:
```suggestion
            'measurement_family' => $attribute->getMetricFamily(),
            'default_measurement_unit' => $attribute->getDefaultMetricUnit(),
```

You should add those 2 properties when type is pim_catalog_metric.
This way, you don't have to update all existing attributes in tests.